### ISSUE TITLE
New lines in anchor's style tag break Autolinker

### ISSUE THREAD

**Issue by philfreo** [gregjacobs/Autolinker.js#79]
If there's a newline (`\n`) in the style tag then bad stuff happens:

```
Autolinker.link('<a href="http://close.io/" style="font-family: Helvetica,\nArial">http://close.io</a>')
```

Yields:

```
<a href="<a href="http://close.io/" target="_blank">close.io</a>" style="font-family: Helvetica,
Arial"><a href="http://close.io" target="_blank">close.io</a></a>
```

If you remove the `\n` then you simply get the expected result of:

```
<a href="http://close.io/" style="font-family: Helvetica, Arial">http://close.io</a>
```

**Comment by gregjacobs** [comment]
Hey, good catch, will take a look.

**Comment by gregjacobs** [comment]
Nice, easy fix in 9cce7f50458102d6d7bbbeca8f2d73c68d337e53/[v0.15.1](https://github.com/gregjacobs/Autolinker.js/releases/tag/0.15.1)

**Comment by philfreo** [comment]
Thanks! :+1:
